CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.0]
+
+### 2025-01-10
+- Feat: Add implementation for `getCurrentPosition`, `watchPosition`, `clearWatch`, `checkPermissions`, and `requestPermissions` on both Android and iOS.
```

Comment:
Just noticed that the two last features are not Cordova-related.

Reply:
That's right, fixed!